sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -638,6 +640,64 @@
     finishTest();
 }
 
+function runUnchangedBufferBindingsTest() {
+    debug("");
+    debug("Testing that buffer bindings cannot be changed while transform feedback is active");
+
+    program = wtu.setupTransformFeedbackProgram(
+        gl, [wtu.simpleVertexShader, wtu.simpleColorFragmentShader], ['gl_Position'], gl.INTERLEAVED_ATTRIBS);
+
+    tf = gl.createTransformFeedback();
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+
+    buf = gl.createBuffer();
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buf);
+
+    gl.beginTransformFeedback(gl.POINTS);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "Transform feedback is active");
+
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, gl.createBuffer());
+    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, "Cannot change the bound buffer while transform feedback is active");
```

Comment:
after this, what happens if you call the bindBufferBase with the unchanging value?
E.g. make the call that would not change the binding?
```
gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buf);
wtu.glErrorShouldBe(gl, gl.NO_ERROR, "unchanging binds succeed");
```
?

Reply:
Updated the test.